apple/swift-docc
# Adds support for comments in warning about extraneous content found after a link in task group item

Bug/issue #, if applicable: #679 
rdar://115321138

## Summary

This adds support for comments after task group list items. When extra content appears after a list item, there is a warning that:
> Extraneous content found after a link in task group list item

However, when a comment is added after a list item, this warning should not occur. This PR adds a check that the extra content is not a comment before creating the warning. 

## Dependencies

N/A

## Testing

Steps:
1. Add a comment in a task group after a link: 
```
- ``link`` 
```
2. Check there are no warnings

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
